### PR TITLE
Add OmotenashiCoin for trezor-firmware

### PR DESCRIPTION
**Pull request by omotenashicoin-project**
Please check our source code.

### CODE REVIEW

**Comment by omotenashicoin-project** [comment]
Please check our source code .
I had single commite.

File: common/defs/coins_details.json [hunk 1]
```diff
@@ -863,6 +863,19 @@
                 }
             ]
         },
+        "bitcoin:MTNS": {
```
Comment (prusnak) [inline common/defs/coins_details.json]: Remove this change.

File: common/defs/support.json [hunk 2]
```diff
@@ -46,6 +46,7 @@
       "bitcoin:ZCL": true,
       "bitcoin:ZEC": true,
       "bitcoin:ZEN": true,
+      "bitcoin:MTNS": true,
```
Comment (prusnak) [inline common/defs/support.json]: Remove this change

File: common/defs/support.json [hunk 3]
```diff
@@ -154,6 +155,7 @@
       "bitcoin:ZEC": "1.7.1",
       "bitcoin:ZEL": "1.8.3",
       "bitcoin:ZNY": "1.8.2",
+      "bitcoin:MTNS": "1.8.2",
```
Comment (prusnak) [inline common/defs/support.json]: Change to 1.8.3

File: common/defs/support.json [hunk 4]
```diff
@@ -1514,6 +1516,7 @@
       "bitcoin:ZEL": "2.1.4",
       "bitcoin:ZEN": "2.0.8",
       "bitcoin:ZNY": "2.1.1",
+      "bitcoin:MTNS": "2.1.1",
```
Comment (prusnak) [inline common/defs/support.json]: Change to 2.1.5

**Comment by omotenashicoin-project** [comment]
I have corrected your indication.
Please review the source code again.

File: common/defs/coins_details.json (hunk 1 above)
Comment (omotenashicoin-project) [reply common/defs/coins_details.json]: I fixed it.

File: common/defs/support.json (hunk 2 above)
Comment (omotenashicoin-project) [reply common/defs/support.json]: I fixed it.

File: common/defs/support.json (hunk 3 above)
Comment (omotenashicoin-project) [reply common/defs/support.json]: I fixed it.

File: common/defs/support.json (hunk 4 above)
Comment (omotenashicoin-project) [reply common/defs/support.json]: I fixed it.

**Comment by omotenashicoin-project** [comment]
Thank you for merging our source code.

**Comment by omotenashicoin-project** [comment]
Hello, prusnak .

We are a Trezor wallet client and have completed testing on the mainnet.
Please consider the official support of OmotenashiCoin's Firmware.

Lightweight Omotenashicoin client URL:
https://github.com/omotenashicoin-project/OmotenashiCoin-HDwalletbinaries/tree/master/stable
Linux
electrum-mtns-3.3.9-x86_64.AppImage
Windows
electrum-mtns-3.3.9.exe

**Pull request merged**